src/components/Heading/test.cypress.tsx

```diff
@@ -51,3 +51,173 @@ describe('Heading', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.viewport(576, 750);
+    cy.contains('This is H4 heading').should('have.css', 'font-size', '20px');
+    cy.viewport(767, 1028);
+    cy.contains('This is H4 heading').should('have.css', 'font-size', '20px');
+    cy.viewport(991, 1028);
+    cy.contains('This is H4 heading').should('have.css', 'font-size', '20px');
+  });
+
+  it('should have Font size 25px for large desktops', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.viewport(992, 1028);
+    cy.contains('This is H4 heading').should('have.css', 'font-size', '25px');
+    cy.viewport(1200, 1028);
+    cy.contains('This is H4 heading').should('have.css', 'font-size', '25px');
+  });
+
+  it('should have Font weight 500 ', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.viewport(576, 1000);
+    cy.contains('This is H4 heading').should('have.css', 'font-weight', '500');
+    cy.viewport(1200, 1028);
+    cy.contains('This is H4 heading').should('have.css', 'font-weight', '500');
+  });
+});
+
+describe('H5 Styles', () => {
+  it('should have Font size 20px for mobile, tablets ,desktops and large desktops', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.viewport(576, 750);
+    cy.contains('This is H5 heading').should('have.css', 'font-size', '20px');
+    cy.viewport(687, 1028);
+    cy.contains('This is H5 heading').should('have.css', 'font-size', '20px');
+    cy.viewport(992, 1028);
+    cy.contains('This is H5 heading').should('have.css', 'font-size', '20px');
+  });
+
+  it('should have Font weight 500 ', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.viewport(576, 1000);
+    cy.contains('This is H5 heading').should('have.css', 'font-weight', '500');
+    cy.viewport(1200, 1028);
+    cy.contains('This is H5 heading').should('have.css', 'font-weight', '500');
+  });
+});
+
+describe('H6 Styles', () => {
+  it('should have Font size 16px for mobile, tablets ,desktops and large desktops', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.viewport(576, 750);
+    cy.contains('This is H6 heading').should('have.css', 'font-size', '16px');
+    cy.viewport(687, 1028);
+    cy.contains('This is H6 heading').should('have.css', 'font-size', '16px');
+    cy.viewport(992, 1028);
+    cy.contains('This is H6 heading').should('have.css', 'font-size', '16px');
+  });
+
+  it('should have Font weight 500 ', () => {
+    mount(<TestThemeWrapper theme={crukTheme}>{content()}</TestThemeWrapper>);
+    cy.viewport(576, 1000);
+    cy.contains('This is H6 heading').should('have.css', 'font-weight', '500');
+    cy.viewport(1200, 1028);
+    cy.contains('This is H6 heading').should('have.css', 'font-weight', '500');
+  });
+});
```

Comment:
isn't this the whole point of snapshots, 

if styles change you don't want to have to be digging into your cypress tests to fix it, you want to approve or disapprove snaps